module/actor-sheet.js

```diff
@@ -698,47 +666,42 @@ async updateAttributes(event) {
     if(!item){
       return;
     }
-    if (item.data[dataset.binding] && item.data[dataset.binding] === input) {
+    if (item.system[dataset.binding] && item.system[dataset.binding] === input) {
       return;
     }
-    else if (item.data.data[dataset.binding] && item.data.data[dataset.binding] === input) {
+    else if (item.system[dataset.binding] && item.system[dataset.binding] === input) {
       return;
     };
 
     await item.update({
       [dataset.binding]: input
     });
 
-    let data = item.data.data;
+    let data = item.system;
     const pool = [0, 1, 1, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 6, 6, 6, 6, 6, 6, 6, 7, 7, 7, 7, 7, 7, 7, 7, 8, 8, 8, 8, 8, 8, 8, 8, 8]
```

Comment:
Here is another place with the pool const that I believe is now unused.  It was used in the commented-out code that I deleted below.

Reply:
Oh, no this one is actually used!  Perhaps add some comments explaining what this is for because it's not obvious.